src/test/resources/com/znsio/teswiz/features/vodqa.feature

```diff
@@ -6,4 +6,12 @@ Feature: Vodqa test
   Scenario: Validating scroll functionality using 2 points
     Given I login to vodqa application using valid credentials
     When I scroll from one to another element point on vertical swiping screen
-    Then Element text should be visible
\ No newline at end of file
+    Then Element text should be visible
+
+#  CONFIG=./configs/vodqa_local_config.properties PLATFORM=android TAG=vodqaContextSwitch ./gradlew run
+  @vodqaContextSwitch @android
+  Scenario: Validate context switching between native and web view context
+    Given I login to vodqa application using valid credentials
+    When I enter into hacker news under webview section
+    Then I am able to see hacker news login option in webview section
```

Comment:
It would be better to merge these 2, as you have done for the last one.
You can name it "I am able to interact with elements on web view section"